lib/modules/manager/types.ts

```diff
@@ -235,6 +235,7 @@ export interface ManagerApi extends ModuleApi {
     content: string,
     currentValue: string,
     bumpVersion: ReleaseType,
+    packageFile?: string,
```

Comment:
```suggestion
    packageFile: string,
```
should not be optional.